submissions/north_sea_mountain.jl

```diff
@@ -0,0 +1,19 @@
+author = "Tim Reichelt"
+description = "North Sea mountain"
+
+using SpeedyWeather, RainMaker
+
+spectral_grid = SpectralGrid(trunc=31, nlayers=8)
+model = PrimitiveWetModel(; spectral_grid)
+model.feedback.verbose = false # hide
```

Comment:
At the moment this line has to be added to prevent the documentation from printing every increment to the progressmeter line by line. This is already fixed in SpeedyWeather by setting `model.feedback.verbose` by default to `isinteractive()` (REPL, notebooks) or `false` for documentation building for example, see https://github.com/SpeedyWeather/SpeedyWeather.jl/pull/582

Reply:
(we just need a new release of SpeedyWeather then this line can be deleted)